src/pages/en/guides/content-collections.mdx

```diff
@@ -234,7 +234,7 @@ defineCollection({
 
 ### Defining custom slugs
 
-Every content entry generates a URL-friendly `slug` property from its file `id`. The slug is used to query the entry directly from your collection. It is also useful when creating new pages and URLs from your content.
+Every content entry generates a URL-friendly `slug` property from its file `id`. The file `id` is simply the filename including its extension (`.md` or `.mdx`), but the extension is stripped for the slug. The slug is used to query the entry directly from your collection. It is also useful when creating new pages and URLs from your content.
```

Comment:
```suggestion
Every content entry generates a URL-friendly `slug` property from its [file `id`](/en/reference/api-reference/#id). The slug is used to query the entry directly from your collection. It is also useful when creating new pages and URLs from your content.
```
Rather than lengthen this paragraph, I think a good immediate measure is to link this to the definition of the propety itself. This reference page I think gives the clearest illustration of the similarity/difference between `slug` and `id`, and avoids extra words for those who don't need it (e.g. are revisiting the page to quickly remind themselves how to do something). Also, when we have a good description of something elsewhere, we do try to link so that if something changes, we only have to update in one place.

Do you think this solves the immediate problem of making it easier to understand what the file `id` is?

Reply:
Yes, this makes perfect sense! I didn't know that API reference page existed or I would have done the same myself. Thank you so much for being awesome. :)